monitoring/api/AlertSnippets/AlertSnippets.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
```

Comment:
nit: remove space

Reply:
I didn't touch the .csproj.  It's 100% purely generated from the dotnet command.  I'd rather leave it that way.